.github/workflows/ci-nix.yml

```diff
@@ -89,6 +89,9 @@ jobs:
           authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
         continue-on-error: true
 
+      - name: Validate codecov.yaml configuration
+        run: curl --fail-with-body -X POST --data-binary @.codecov.yml https://codecov.io/validate
```

Comment:
Do we need to add curl to flake.nix? Or `nix run nixpkgs#curl` ;)

Reply:
You're thinking like a Nixer! :+1: